### PR TITLE
imagedata tag when is inside an object tag was not parsed

### PR DESCRIPTION
**Pull request by IuryAlves**
### when there was an imagedata tag inside an object tag , the imagedata tag was not being parsed:

```
<w:object>
  <v:shape id="ole_rId2" style="width:108.7pt;height:19pt">
    <v:imagedata r:id="rId3" o:title=""/>
  </v:shape>
</w:object>
```
**Note: I try to write tests for this commit, but i had no sucess. Maybe someone can point me in the right direction.**

### CODE REVIEW

**Comment by kylegibson** [comment]
Hi Lury!

Thanks so much for this pull request. I'm on my way to bed, so I'll review it tomorrow as soon as I am able!

-Kyle

**Comment by IuryAlves** [comment]
@kylegibson  Me too = D . Ok, good night.

File: pydocx/export/base.py [hunk 1]
```diff
@@ -137,7 +137,8 @@ def _convert_complex_fields_into_simple_fields(self):
                     if previous_run.parent is not run.parent:
                         # scope has changed
                         fields.append(field)
-                        field = wordprocessing.SimpleField(instr=field.instr, children=[])
+                        field = wordprocessing.SimpleField(
+                            instr=field.instr, children=[])
```
Comment (kylegibson) [inline pydocx/export/base.py]: Why was this changed?

File: pydocx/export/base.py [hunk 2]
```diff
@@ -265,7 +266,8 @@ def export_paragraph(self, paragraph):
         children = self.yield_paragraph_children(paragraph)
         results = self.yield_nested(children, self.export_node)
         if paragraph.effective_properties:
-            results = self.export_paragraph_apply_properties(paragraph, results)
+            results = self.export_paragraph_apply_properties(
+                paragraph, results)
```
Comment (kylegibson) [inline pydocx/export/base.py]: Why was this changed?

File: pydocx/export/base.py [hunk 3]
```diff
@@ -441,6 +443,9 @@ def export_footnote_reference_mark(self, footnote_reference_mark):
     def export_vml_shape(self, shape):
         return self.yield_nested(shape.children, self.export_node)
 
+    def export_object(self, _object):
```
Comment (kylegibson) [inline pydocx/export/base.py]: How do you feel about naming this `obj` so we can avoid the underscore?

File: pydocx/openxml/wml/object.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    absolute_import,
+    print_function,
+    unicode_literals,
+)
+
+from pydocx.models import XmlModel, XmlCollection
+from pydocx.openxml.vml.shape import Shape
+
+
+class Object(XmlModel):
```
Comment (kylegibson) [inline pydocx/openxml/wml/object.py]: Based on my investigation, it looks like `w:object` refers to this structure:

https://msdn.microsoft.com/en-us/library/documentformat.openxml.wordprocessing.embeddedobject%28v=office.15%29.aspx

I think this should be `EmbeddedObject` and be in the `wordprocessing` module. Thoughts?

File: pydocx/export/base.py [hunk 5]
```diff
@@ -472,7 +477,8 @@ def export_numbering_item(self, numbering_item):
         return self.yield_nested(numbering_item.children, self.export_node)
 
     def export_simple_field(self, simple_field):
-        default_results = self.yield_nested(simple_field.children, self.export_node)
+        default_results = self.yield_nested(
+            simple_field.children, self.export_node)
```
Comment (kylegibson) [inline pydocx/export/base.py]: Why was this changed?

File: pydocx/export/base.py (hunk 1 above)
Comment (IuryAlves) [inline pydocx/export/base.py]: pep8

File: pydocx/export/base.py (hunk 2 above)
Comment (IuryAlves) [inline pydocx/export/base.py]: pep8 too

File: pydocx/export/base.py (hunk 3 above)
Comment (IuryAlves) [inline pydocx/export/base.py]: Ok. I will change.

File: pydocx/openxml/wml/object.py (hunk 4 above)
Comment (IuryAlves) [inline pydocx/openxml/wml/object.py]: Make sense. I didn't realize that [wml](http://www.datypic.com/sc/ooxml/ns-w.html) is the same that wordprocessing.

File: pydocx/export/base.py (hunk 5 above)
Comment (IuryAlves) [inline pydocx/export/base.py]: pep8, perhaps my pep8 version is different of yours, i use 1.5.7

File: pydocx/openxml/wordprocessing/__init__.py [hunk 6]
```diff
@@ -39,8 +39,10 @@
 from pydocx.openxml.wordprocessing.table_cell_properties import TableCellProperties  # noqa
 from pydocx.openxml.wordprocessing.table_row import TableRow
 from pydocx.openxml.wordprocessing.text import Text
+from pydocx.openxml.wordprocessing.embedded_object import EmbeddedObject
 
 __all__ = [
+    'EmbeddedObject',
```
Comment (kylegibson) [inline pydocx/openxml/wordprocessing/__init__.py]: Could you put this in sorted order?

File: pydocx/openxml/wordprocessing/__init__.py [hunk 7]
```diff
@@ -39,8 +39,10 @@
 from pydocx.openxml.wordprocessing.table_cell_properties import TableCellProperties  # noqa
 from pydocx.openxml.wordprocessing.table_row import TableRow
 from pydocx.openxml.wordprocessing.text import Text
+from pydocx.openxml.wordprocessing.embedded_object import EmbeddedObject
```
Comment (kylegibson) [inline pydocx/openxml/wordprocessing/__init__.py]: Could you put this in sorted order?

File: pydocx/openxml/wordprocessing/embedded_object.py [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+    unicode_literals,
+)
+
+from pydocx.models import XmlModel, XmlCollection
+from pydocx.openxml.vml.shape import Shape
+
+
+class EmbeddedObject(XmlModel):
+    """
+    reference: http://www.datypic.com/sc/ooxml/e-w_object-1.html
```
Comment (kylegibson) [inline pydocx/openxml/wordprocessing/embedded_object.py]: I think we should reference https://msdn.microsoft.com/en-us/library/documentformat.openxml.wordprocessing.embeddedobject%28v=office.15%29.aspx instead or additionally. What do you think?

File: pydocx/export/base.py (hunk 5 above)
Comment (kylegibson) [inline pydocx/export/base.py]: I have pep8 1.5.7 installed too. I also have flake8 2.4.1, which is what travis installs each time the tests are run. flake8 reads the `[flake8]` section in `tox.ini` which has `max-line-length = 95`.

File: pydocx/openxml/wordprocessing/embedded_object.py (hunk 8 above)
Comment (IuryAlves) [inline pydocx/openxml/wordprocessing/embedded_object.py]: alright

**Comment by IuryAlves** [comment]
This solves #179

**Comment by kylegibson** [comment]
Awesome! Thank you!

**Comment by IuryAlves** [comment]
Sure, no prob! ;)
